src/tests/services/project-manager.test.js

```diff
@@ -97,16 +104,21 @@ describe('service project manager', () => {
               name
               projectManager {
                 _id
-                fullName
+                firstName
+                lastName
               }
             }
           }
         }
       `
     })
-      .expect(res => {
-        expect(res.body).toHaveProperty('data.createOneService.record.projectManager')
-        expect(res.body.data.createOneService.record.projectManager.fullName).toStrictEqual(newProjectManager.fullName)
+      .expect((res) => {
+        expect(res.body).toHaveProperty(
+          'data.createOneService.record.projectManager'
+        )
+        expect(
+          res.body.data.createOneService.record.projectManager.fullName
```

Comment:
Update this test since you've dropped the `fullName` in favor of `firstName` and `lastName`